src/views/List.jsx

```diff
@@ -1,34 +1,36 @@
-import { useEffect, useState } from 'react';
+import { useState } from 'react';
 import { ListItem } from '../components';
-import { getItemData, streamListItems } from '../api';
 
-export function List({ listToken }) {
-	const [data, setData] = useState([]);
-	const [loading, setLoading] = useState(true);
+export function List({ data }) {
+	const [searchField, setSearchField] = useState('');
 
-	useEffect(() => {
-		streamListItems(listToken, (snapshot) => {
-			const nextData = getItemData(snapshot);
+	const onSearchChange = (e) => {
+		const { value } = e.target;
+		setSearchField(value.toLocaleLowerCase());
```

Comment:
Nice thinking here of converting the search value to lowercase here :+1: 
Unless you are planning to support different locales in the future, you can just use `toLowerCase` here.

Reply:
Got it!